src/components/HomeCard/home-card.styles.jsx

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
+import Container from '../../utils/components/Container';
+
+
+export const Circle = styled.div`
+    height: 30px;
+    width: 30px;
+    color: #fff;
+    background-color: #6831DE;
+    border-radius: 100%;
+    font-weight: bold;
+    font-family: 'Poppins', sans-serif;
```

Comment:
Yes, I hardcoded it because I wanted the number inside each Circle to have the Poppins font-family. If I removed it, the number would have a default font-family of ``sans-serif``

In ``home-card.component.jsx`` I added the Poppins font family prop to the Text component, which is the description below the Icon.